### PR TITLE
Avoid ES errors with default shell settings

### PR DESCRIPTION
**Pull request by stevendanna**
Elasticsearch by default expects at least 65535 potential open files to be available from the kernel through the shell process it starts up in.

Signed-off-by: Steven Danna <steve@chef.io>

### CODE REVIEW

**Comment by vjeffrey** [comment]
rebasing on master and force pushing to get this merged in

File: components/automate-chef-io/content/docs/system-requirements.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ Chef Automate requires
 * `systemd` as the init system
 * `useradd`
 * `curl` or `wget`
+* The shell that starts Automate should have a # of open files max of at least 65535, 
+not 1024 which is common
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/system-requirements.md]:
```suggestion
```

File: components/automate-chef-io/content/docs/system-requirements.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ Chef Automate requires
 * `systemd` as the init system
 * `useradd`
 * `curl` or `wget`
+* The shell that starts Automate should have a # of open files max of at least 65535,
```
Comment (mjingle) [inline components/automate-chef-io/content/docs/system-requirements.md]:
```suggestion
* a shell that starts Automate with an open files max setting of 65535 or more. (1024 is the common default setting.)
```

Comment (mjingle) [reply components/automate-chef-io/content/docs/system-requirements.md]: Please check me on if I've phrased this correctly.

**Pull request merged**